index.js

```diff
@@ -86,7 +87,17 @@ class Equivalency {
     });
 
     let isEquivalent = s1prime === s2prime;
-    return { isEquivalent: isEquivalent };
+
+    let results = { isEquivalent: isEquivalent };
+
+    if (options && options.calculateEditDistance) {
+      const editDistance = dl(s1prime, s2prime);
+      results = Object.assign({}, results, {
+        editDistance: editDistance.steps,
+      });
```

Comment:
Originally tried a `{ ...results, editDistance: editDistance.steps };` step here, but the spread syntax was breaking eslint.

Reply:
William's solution looks like the way to go here, but for future reference, adding this block to `.eslintrc` lets eslint know that spread syntax is okay. I'd first check that it actually is okay before adding this, but it usually is.
```
﻿  "parserOptions": {
    "ecmaVersion": 8,
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true
    }
  }
```

Reply:
Thanks for that! Didn't want to mess with the linter but I'll ping the group the next time I run into something like this :)